src/applications/claims-status/containers/YourClaimsPageV2.jsx

```diff
@@ -232,29 +236,51 @@ class YourClaimsPageV2 extends React.Component {
 
     return (
       <>
-        <div name="topScrollElement" />
+        {!this.props.isSmoothLoadingEnabled && <div name="topScrollElement" />}
         <article className="row">
           <div className="usa-width-two-thirds medium-8 columns">
-            <ClaimsBreadcrumbs />
+            <div
+              className={`${
+                this.props.isSmoothLoadingEnabled ? 'breadcrumbs-container' : ''
+              }`}
+            >
+              <ClaimsBreadcrumbs />
+            </div>
             <h1 className="claims-container-title">
               Check your claim, decision review, or appeal status
             </h1>
-            <va-on-this-page />
+            <div
+              className={`${
+                this.props.isSmoothLoadingEnabled
+                  ? 'on-this-page-container'
+                  : ''
+              }`}
+            >
+              <va-on-this-page />
+            </div>
             <h2 id="your-claims-or-appeals" className="vads-u-margin-top--2p5">
               Your claims, decision reviews, or appeals
             </h2>
             <div>{this.renderErrorMessages()}</div>
-            <va-additional-info
-              id="claims-combined"
-              class="claims-combined"
-              trigger="Find out why we sometimes combine claims."
+            <div
+              className={`${
+                this.props.isSmoothLoadingEnabled
+                  ? 'additional-info-container'
```

Comment:
There appears to be a race condition combined with a class name clash related to createAdditionalInfoWidget() on slower network speeds (e.g., 3G).

https://github.com/user-attachments/assets/e71d2eff-aa7a-4f3d-a912-fb1bc9b374f5

Changing the class name to something like `additional-info-component-container` will resolve this issue.